lib/devices/android/android-hybrid.js

```diff
@@ -101,9 +101,15 @@ androidHybrid.getProcessNameFromWebview = function (webview, cb) {
     _.find(lines, function (line) {
       line = line.trim().split(/\s+/);
       if (line[pidColumn].indexOf(pid) !== -1) {
-        logger.debug("Parsed pid: " + line[pidColumn] + " pkg: " + line[pkgColumn]);
-        logger.debug("from: " + line);
-        pkg = line[pkgColumn];
+        if (typeof line[pkgColumn] != 'undefined') {
```

Comment:
These can be consolidated if you keep track of the index only, rather than doing everything in the `if/then`.

```js
if (typeof line[pkgColumn] === 'undefined') {
  pkgColumn = pkgColumn - 1;
}
pkg = line[pkgColumn];
logger.debug("Parsed pid: " + line[pidColumn] + " pkg: " + pkg);
logger.debug("from: " + line);
```